test/presenters/hotwire_combobox/component_test.rb

```diff
@@ -0,0 +1,51 @@
+require 'test_helper'
+
+class HotwireCombobox::ComponentTest < ApplicationViewTestCase
+  setup do
+    @view = ActionView::Base.new(ActionController::Base.view_paths, {}, ActionController::Base.new)
+  end
+
+  test "hidden_field_attrs returns value from form object if available" do
+    existent_field_on_model = "Peter"
+    field_name = :name
+    component = HotwireCombobox::Component.new(
+      @view, field_name,
+      form: mock_form(form_object: OpenStruct.new("#{field_name}": existent_field_on_model)),
+    )
+
+    assert_equal component.hidden_field_attrs, { id: "#{field_name}_id-hw-hidden-field",
```

Comment:
Instead of asserting all attrs, let's just assert that the attr hash includes the correct name and value.